frontend/sw360-portlet/src/main/webapp/html/moderation/releases/delete.jsp

```diff
@@ -52,13 +52,8 @@
 <div id="header"></div>
 <p class="pageHeader"><span class="pageHeaderBigSpan">Moderation Delete Release: <sw360:ReleaseName release="${release}" /></span>
 </p>
-<input type="button" onclick="acceptDelete()" id="edit" value="Accept Delete"    class="acceptButton">&nbsp;
-<input type="button" onclick="removeFromModerators()" id="edit" value="Remove Me from Moderators"    class="ignoreButton">&nbsp;
-<input type="button" onclick="declineDelete()" id="edit" value="Decline Deletion"    class="addButton">&nbsp;
-<input type="button" onclick="postPone()" id="edit" value="Postpone Decision"    class="postponeButton">&nbsp;
-<input type="button" onclick="cancel()" id="edit" value="Cancel"    class="cancelButton">
-
-
+<%@include file="/html/moderation/includes/moderationActionButtons.jspf"%>
+<br/>
```

Comment:
This return looks like a hack to solve a css problem. Might be better to wrap the buttons into a div and use the `display:...` or `margin:...` properties